scalding-core/src/main/scala/com/twitter/scalding/reducer_estimation/RatioBasedEstimator.scala

```diff
@@ -48,6 +48,9 @@ abstract class RatioBasedEstimator extends ReducerEstimator {
     val threshold = RatioBasedEstimator.getInputRatioThreshold(conf)
 
     historyService.fetchHistory(info, maxHistory) match {
+      case Success(Nil) =>
```

Comment:
It returns a Try[Seq[FlowStepHistory]].
Scala Seqs match by contents, so List() == Seq(), but yeah I guess it's possible for that to change.